document_similarity/microservice/routes/service.py

```diff
@@ -193,11 +190,17 @@ def get_similarities():
         # TODO: change the expression if needed
         # get only the lines in table 'similarities' where the first document has id doc_id
         # sort them by the similarity column, descending
-        similarity_list = pg.execute("""
-            SELECT document2_id, similarity_score FROM similarities
-            WHERE document1_id = {}
-            ORDER BY similarity_score DESC;
-            """.format(doc_id))
+
+        # SQL constraints:
+        constraints = """WHERE document1_id = {}
```

Comment:
When using tripple quotes, move the string into the next line (as seen in the removed code block above